go.mod

```diff
@@ -34,7 +34,6 @@ require (
 	github.com/hashicorp/go-memdb v1.0.3
 	github.com/hashicorp/go-msgpack v0.5.5
 	github.com/hashicorp/go-multierror v1.0.0
-	github.com/hashicorp/go-plugin v1.0.1
```

Comment:
I dont see any `go.sum` change. Did you run `go mod tidy`?

Reply:
Apparently not. Fixing...

Reply:
Actually I did - there is no diff. I think IIRC the rationale is that go.sum always remembers versions of deps even after you stop using them or security reasons in the future. See https://github.com/golang/go/issues/26381 it's treated as "append only".